ktor-client/ktor-client-okhttp/jvm/src/io/ktor/client/engine/okhttp/OkHttpEngine.kt

```diff
@@ -39,14 +49,22 @@ class OkHttpEngine(override val config: OkHttpConfig) : HttpClientEngineBase("kt
         builder.build()
     }
 
+    /**
+     * Cache that keeps least recently used [OkHttpClient] instances.
+     */
+    private val clientCache = createLRUCache(::createOkHttpClient, {}, CLIENT_CACHE_SIZE)
```

Comment:
Consider adding size to engine configuration, 0 for no cache.

Reply:
Done.